utils/build/docker/dotnet_http/parametric/ApmTestApi.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <InvariantGlobalization>true</InvariantGlobalization>
+    <UserSecretsId>f56b479c-4bfa-4286-bb36-96a6273695e3</UserSecretsId>
+    <Configuration Condition="'$(Configuration)' == ''">Development</Configuration>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+    <PackageReference Include="Datadog.Trace" Version="*" Condition="'$(Configuration)' != 'Release'" />
```

Comment:
I like the idea here, but I don't think we should tie this to the `Configuration`. What if I want to do a `Release` build locally? (Isn't that the default nowadays anyway?)

Instead, can we use a new msbuild property with a sensible default that we can override? Something like

```xml
<PackageReference Include="Datadog.Trace" Version="*" Condition="'$(USE_DATADOG_TRACE_NUGET)' == 'true'" />
```

Reply:
To clarify, this is useful when making changes to the project and getting Intellisense to work and building locally
After you do that though, you have to delete the the `bin`/`obj` directories (even though they are excluded in the `.dockerignore` not really sure why)

We could do something like that but the reason to include the `PackageReference` is just to get Intellisense support, so it's not really to "use" it as an actual NuGet

I'd be fine making it a bit more explicit just so it doesn't accidentally get included though with something like that though

Reply:
So I didn't get issues with bin/obj with this project as I did with the old one, will need to play around with it to see which option matches better, lets see how it runs after rolling back the new folder change.

Reply:
> To clarify, this is useful when...

No, I get it. I think it's a good idea! I also add the nuget package locally when working on parametric tests (well, since it was removed 😉 ), I just never push that change.

My issue is with trying this to `Condition="'$(Configuration)' != 'Release'"`. I should be able to choose to build either debug or release locally for different reasons without breaking this nuget include.

Reply:
Ahh gotcha
Yeah that makes sense, so yeah it'd probably be best to wrap this with `USE_DATADOG_TRACE_NUGET` instead of the build configuration